_applications/tensorflow_mnist.adoc

```diff
@@ -0,0 +1,76 @@
+= Tensorflow MNIST
+:page-link: tensorflow_mnist
+:page-weight: 100
+:page-labels: [Tensorflow, S2I]
+:page-layout: application
+:page-menu_template: menu_tutorial_application.html
+:page-description: This demo shows how to use source-to-image Tensorflow Serving build to deploy a tensorflow serving prediction endpoint on Openshift. The s2i build provides a GRPC microservice endpoint for web applications to send queries to be evaluated against the tensorflow model.
+:page-project_links: ["https://github.com/radanalyticsio/tensorflow-serving-s2i"]
+
+[[introduction]]
+== Introduction
+
+MNIST digit identification application provides an example of how to use s2i build to deploy a tensorflow prediction endpoint on Openshift.
+Input data is taken from  http://yann.lecun.com/exdb/mnist/[MNIST site].
+
+
+[[architecture]]
+== Architecture
+
+The system is composed of:
+
+- A front-end application: Which has a canvas where user can draw a digit and then the image is sent to 2 prediction endpoints.
+- A Prediction Endpoint (regression): The tensorflow serving endpoint which serves a https://github.com/radanalyticsio/tensorflow-serving-s2i/tree/master/models/regression/1[regression model].
+- A Prediction Endpoint (cnn): The tensorflow serving endpoint which serves a https://github.com/radanalyticsio/tensorflow-serving-s2i/tree/master/models/cnn/1[CNN model].
+
+[[installation]]
+== Installation
+
+Installation of this application is using the `oc` command line
+tool. Once you are logged in to your OpenShift cluster, follow these
+instructions:
+
+
+1. Ensure that you are connected to an OpenShift project
+	
+	oc new-project test
+
+2. Create the template for mnist web application.
+
+   oc create -f https://raw.githubusercontent.com/sub-mod/mnist-app/master/template.json
```

Comment:
Subin, this is great.  My only quibble is this:  should we host `mnist-app` under the organization account?

Reply:
ok, fine with me to take it as an action to fix in the near future